pyglet/graphics/vertexdomain.py

```diff
@@ -103,6 +104,18 @@ def __init__(self, program, attribute_meta):
         self.attribute_names = {}
         for attribute in self.attributes:
             self.attribute_names[attribute.name] = attribute
+        
+        self._create_attribute_buffer_arrays()
+
+    def _create_attribute_buffer_arrays(self):
+        """
+        For each attribute, create a ctypes array pointing to the entire buffer
+        which can be used to write values into the buffer using slice assignment
+        """
```

Comment:
Minor style nitpick: pyglet's docstring conventions place the summary on the same line as the opening quotes.
```suggestion
        """For each attribute, create a ctypes array pointing to to its entire buffer.

        This allows using slice assignment syntax to write values into the buffer.
        """
```